### PR TITLE
Update contributors for 0.1.61 release

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Update list of contributors for 0.1.61 release.

#### Rationale:

- Thank you all for your contributions!

### CODE REVIEW

File: ssg/contributors.py [hunk 1]
```diff
@@ -42,6 +44,8 @@
     # Joseph D Lenox
     "jdlenox@labnet.rockwellcollins.lab": "Joseph.Lenox@collins.com",
     "lordofhyphens@gmail.com": "Joseph.Lenox@collins.com",
+    "Joseph.Lenox@corp.rockwellcollins.com": "Joseph.Lenox@collins.com",
+    "joseph.lenox@collins.com": "Joseph.Lenox@collins.com",
```
Comment (yuumasato) [inline ssg/contributors.py:48]: @lenox-joseph Hi, do you have a preferred name and email to put in the contributors list?

Comment (lenox-joseph) [reply ssg/contributors.py:48]: @yuumasato
Joseph Lenox
joseph.lenox@collins.com

Comment (yuumasato) [reply ssg/contributors.py]: Updated.

Comment (lenox-joseph) [reply ssg/contributors.py]: > Updated.

Thank you. Sorry about having the pile of associated email addresses.

File: Contributors.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ The following people have contributed to the SCAP Security Guide project
 * Lenox, Joseph D (Collins) <Joseph.Lenox@collins.com>
 * 0x66656c6978 <0x66656c6978@users.noreply.github.com>
 * Jack Adolph <jack.adolph@gmail.com>
+* Edgar Aguilar <edgar.aguilar@oracle.com>
```
[the remark this answers is not in the document]
Comment (Xeicker) [reply Contributors.md:11]: This looks good

**Pull request merged**